src/modules/launcher/PowerLauncher/ViewModel/MainViewModel.cs

```diff
@@ -828,7 +828,7 @@ protected virtual void Dispose(bool disposing)
                         _hotkeyManager.UnregisterHotkey(_hotkeyHandle);
                     }
                     _hotkeyManager.Dispose();
```

Comment:
What about _hotkeyManager? Can it also be problematic?

Reply:
Maybe. Certainly adding a `?` there won't do any harm.